pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -113,10 +113,25 @@ func (c *cyclonedxParser) addRootPackage(cdxBom *cdx.BOM) {
 		} else {
 			splitImage := strings.Split(cdxBom.Metadata.Component.Name, "/")
 			if len(splitImage) == 3 {
-				rootPackage.Purl = "pkg:oci/" + splitImage[2] + "?repository_url=" + splitImage[0] + "/" + splitImage[1]
-				rootPackage.Version = cdxBom.Metadata.Component.Version
-				rootPackage.Digest = append(rootPackage.Digest, cdxBom.Metadata.Component.Version)
-				rootPackage.Tags = []string{"CONTAINER"}
+				// example: gcr.io/distroless/static:nonroot
+				splitTag := strings.Split(splitImage[2], ":")
```

Comment:
may need to check if this exists, if tag doesn't exist then may not need to include. can include 1 test with no tag as well